instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/patches/base.rb

```diff
@@ -20,6 +20,7 @@ def initialize(*args)
             @metadata = {}
             super
           end
+          ruby2_keywords(:initialize) if respond_to?(:ruby2_keywords, true)
```

Comment:
I was curious if using `double splat` would be enough, i.e. `def initialize(**kwargs)`, but it looks like this is how it is defined in `rails/active_job/core`:

https://github.com/rails/rails/blob/53d54694e9a423642ba9c984207097e9522db26f/activejob/lib/active_job/core.rb#L100

It looks like once we have minimal support for 2.7 we could remove the `respond_to` check.
Is there a min version of `ActiveJob`, and by extension Ruby, that we are looking to support for this instrumentation?

Reply:
@arielvalentin 5.2 is the lowest supported version: https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/instrumentation.rb#L12

Reply:
I *think* double splat will only capture keyword args and not positional args, but I could be wrong about that. I do think it makes sense to mirror what canonical activejob is doing as closely as possible.